app/views/people/show.html.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Task Master Details</h1>
+
+<h2>Name: <%= @name %> </h2>
+<h2>Occupation: <%= @occupation %> </h2>
+<h2>Fave Color: <%= @color %> </h2>
+<h3> <%= link_to "View #{@name}'s Tasks", "/people/#{@person[:id]}/tasks", id: "add", method: :get %> </h3>
```

Comment:
Even though one of these links is only ever displayed on a single page,  you want to be careful with using a static CSS ID because this can cause issues if there are multiple items on a single page with the same ID.